src/test/resources/assets/modid/bc/MMPL-1.0.txt.txt

```diff
@@ -0,0 +1,90 @@
+Minecraft: Denotes a copy of the Minecraft game licensed by Mojang AB
+
+User: Anybody that interacts with the software in one of the following ways:
+   - play
+   - decompile
+   - recompile or compile
+   - modify
+   - distribute
+
+Mod: The mod code designated by the present license, in source form, binary
+form, as obtained standalone, as part of a wider distribution or resulting from
+the compilation of the original or modified sources.
+
+Dependency: Code required for the mod to work properly. This includes 
+dependencies required to compile the code as well as any file or modification
+that is explicitely or implicitely required for the mod to be working.
+
+1. Scope
+--------
+
+The present license is granted to any user of the mod. As a prerequisite, 
+a user must own a legally acquired copy of Minecraft
+
+2. Liability
+------------
+
+This mod is provided 'as is' with no warranties, implied or otherwise. The owner
+of this mod takes no responsibility for any damages incurred from the use of
+this mod. This mod alters fundamental parts of the Minecraft game, parts of
+Minecraft may not work with this mod installed. All damages caused from the use
+or misuse of this mad fall on the user.
+
+3. Play rights
+--------------
+
+The user is allowed to install this mod on a client or a server and to play 
+without restriction.
+
+4. Modification rights
+----------------------
+
+The user has the right to decompile the source code, look at either the 
+decompiled version or the original source code, and to modify it.
+
+5. Derivation rights
+--------------------
+
+The user has the rights to derive code from this mod, that is to say to
+write code that extends or instanciate the mod classes or interfaces, refer to
+its objects, or calls its functions. This code is known as "derived" code, and 
+can be licensed under a license different from this mod.
+
+6. Distribution of original or modified copy rights
+---------------------------------------------------
+
+Is subject to distribution rights this entire mod in its various forms. This
+include:
+   - original binary or source forms of this mod files
+   - modified versions of these binaries or source files, as well as binaries
+     resulting from source modifications
+   - patch to its source or binary files
+   - any copy of a portion of its binary source files
+
+The user is allowed to redistribute this mod partially, in totality, or 
+included in a distribution.
+
+When distributing binary files, the user must provide means to obtain its 
+entire set of sources or modified sources at no costs.
+
+All distributions of this mod must remain licensed under the MMPL.
+
+All dependencies that this mod have on other mods or classes must be licensed
+under conditions comparable to this version of MMPL, with the exception of the
+Minecraft code and the mod loading framework (e.g. ModLoader, ModLoaderMP or
+Bukkit).
+
+Modified version of binaries and sources, as well as files containing sections
```

Comment:
Why the crayon licence

Reply:
Not neccesarily a problem - just an explanation please

Reply:
The license needs to be reproduced alongside buildcraft assets. Perhaps the bc textures could be replaced with some cc0 textures?

Reply:
That's just what the old BuildCraft assets/code is licensed under. It was the most readily available texture atlas I had around which was in any way helpfully licensed, the textures themselves are irrelevant so you're welcome to find/make a more freely licensed alternative to these.